src/components/modules/publication.js

```diff
@@ -0,0 +1,56 @@
+import React, { useMemo, useState } from 'react'
+import { Module } from '../layout'
+import styled from 'styled-components'
+
+const Toggler = styled.button.attrs({ role: 'button' })`
+  width: 9rem;
+  height: 2rem;
+  font-weight: 600;
+  background-color: transparent;
+  border: 2px var(--color-primary) solid;
+  border-radius: 5px;
+  cursor: pointer;
+  color: var(--color-primary);
+  transition: color 250ms;
+  &:hover {
+    color: var(--color-primary-dark);
+  }
+`
+
+export const Publication = props => {
```

Comment:
accessibility should be addressed here. check out the ARIA attributes `aria-controls`, `aria-expanded`, and `aria-labelled-by`.

Reply:
Thanks for reminding about the accessibility. I'll add aria arributes.